tests/compas/geometry/test_basics.py

```diff
@@ -34,7 +33,7 @@
 ]
 )
 def test_angle_vectors(u, v, angle):
-    assert angle_vectors(u, v) == pytest.approx(angle)
+    assert close(angle_vectors(u, v), angle)
```

Comment:
so you prefer that we rely on our own interpretation of "close enough"?

Reply:
Not really, again and just like the change of `np.allclose` to `compas.geometry.allclose`, the idea is was only to make the tests run. In this case, using `pytest.approx` was not really adding a lot of benefit than what we could easily do with our `close`, and implementing `approx` in my pytest clone would have been too much work (it seems to rely on a lot of magic methods to do its magic).